modules/ROOT/pages/rtf-deployment-settings.adoc

```diff
@@ -43,19 +43,19 @@ When users replay transmissions when Partner Manager is deployed to Runtime Fabr
 | Application Name Prefix |  Prefix appended to the Partner Manager application names. Maximum length is five characters. | 
 | Each B2B endpoint | Reserved CPU space, CPU space limit, and memory capacity for each Partner Manager endpoint. Partner Manager deploys a generic Mule app to the runtime for each endpoint protocol and direction (receive or send) combination. For help with sizing, contact your MuleSoft Account Team. a|
 
-* *Reserved CPU*: 80m
-* *CPU Limit*: 500m
-* *Memory*: 900Mi
+* *Reserved CPU*: .08 vCPU
+* *CPU Limit*: .5 vCPU
```

Comment:
optional but maybe change these decimal numbers to include the 0 at the beginning to make it more clear. Like instead of .5 -> 0.5 so it doesn't look like 5

Reply:
Good idea but I am keeping as is to reflect what's in the GUI.